sdk/python/src/buildkite_sdk/types.py

```diff
@@ -135,24 +141,58 @@ def __init__(
             pagerduty_change_event=None,
         )
 
+
+class NotifyGithubCommitStatus(_purple_github_commit_status):
```

Comment:
Similar note about the GitHub `H` thing here also (sorry -- just speaking from experience on this one!):

```suggestion
class NotifyGitHubCommitStatus(_purple_github_commit_status):
```